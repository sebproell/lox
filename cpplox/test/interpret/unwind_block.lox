var a = 1;
{
    print a + b;
    // This should never happen
    a = 2;
}
print a;
