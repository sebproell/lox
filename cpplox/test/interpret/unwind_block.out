Undefined variable 'b'.
[line 3]
1
